### PR TITLE
Ch20120/collapsable folder

### PR DESCRIPTION
**Pull request by SamueleA**
-Added collapsable folders

Peek 2020-11-24 15-15

### CODE REVIEW

**Comment by morochroyce** [comment]
nice, will be good to go once ripple is disabled.

**Review by morochroyce — APPROVED** [review]

**Comment by SamueleA** [comment]
Remove ripple effects when clicking on the arrows in 7429fb1

File: src/shared/components/ObjectsTable/ObjectsTable.js [hunk 1]
```diff
@@ -192,6 +210,28 @@ const ObjectsTable = ({
...
+    const expanded = !clickedRow.expanded;
+    const newRows = [
+      ...sortedRows.slice(0, rowIndex),
+      {
+        ...clickedRow,
+        expanded,
+      },
+      ...sortedRows.slice(rowIndex + 1, sortedRows.length),
+    ];
+
```
Comment (mateuszJS) [inline src/shared/components/ObjectsTable/ObjectsTable.js:224]:
```
    const newRows = [
      {
        ...clickedRow,
        expanded,
      },
    ];
```
wouldn't be enough?

Comment (mateuszJS) [reply src/shared/components/ObjectsTable/ObjectsTable.js:224]: or action `UPDATE_OR_ADD_OBJECT` with object (without wrapping in an array) as a payload
```
      {
        ...clickedRow,
        expanded,
      },
```

**Review by mateuszJS — APPROVED** [review]
LGTM, suggestion to reduce payload passed in redux action

File: src/shared/components/ObjectsTable/ObjectsTable.js (hunk 1 above)
Comment (SamueleA) [reply src/shared/components/ObjectsTable/ObjectsTable.js:221]: Ah you're right, the redux action already filters the array so no need to pass everything back.
Fixed in 84f1ca3

**Pull request merged**